tests/director/api/fetch_request_test.go

```diff
@@ -0,0 +1,70 @@
+package api
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+)
+
+const OpenAPISpec = "https://raw.githubusercontent.com/kyma-incubator/github-slack-connectors/master/github-connector/internal/registration/configs/githubopenAPI.json"
```

Comment:
I know we agreed early that we can use file from github, but it may be problematic when moving repo / github outages. To increase resiliency of this test, what do you think about running httptest server and serve the spec? It shouldn't take much time to do it.

Reply:
On the other hand, a file that's actually hosted outside the cluster may be better. I don't github outages are that frequent, and I'm pretty sure we did something similar in Asset Store tests, right?

Reply:
> On the other hand, a file that's actually hosted outside the cluster may be better

I think the most option would be to host it locally with local server. It doesn't matter then if this is a cluster of not. Please remember that these tests are not integration tests run in a separate pod on cluster.

Reply:
Ah, right - this is integration test, so that's absolutely fine. Sorry for confusion